### PR TITLE
Feat: add ips create function

### PR DESCRIPTION
**Pull request by SHA888**
- Added IPS collection create function
- Added IPS class create function
- Added IPS create function

### CODE REVIEW

**Comment by arrudagates** [comment]
Can we solve #11 before merging this? Then we can rebase and use the checks.

**Comment by SHA888** [comment]
Will check it. Thanks @arrudagates

File: ips/src/lib.rs [hunk 1]
```diff
@@ -23,96 +23,124 @@
 
 pub use pallet::*;
 
-#[cfg(test)]
-mod tests;
+// #[cfg(test)]
+// mod tests;
+
+// IpsId
```
Comment (arrudagates) [inline ips/src/lib.rs:29]: Make these comments documentation comments (`///` instead of `//`)

File: ips/src/lib.rs [hunk 2]
```diff
@@ -23,96 +23,124 @@
...
+pub type IpsId = u32;
+pub struct Ips {
+    pub name: Vec<u8>,
+    pub description: Vec<u8>,
+    // Metadata from ipfs
```
Comment (arrudagates) [inline ips/src/lib.rs:34]: Same as above

File: ips/src/lib.rs [hunk 3]
```diff
@@ -23,96 +23,124 @@
...
+    pub properties: Vec<u8>,
+}
+
+pub struct IpsData<Balance> {
+    //Deposit balance to create each token
```
Comment (arrudagates) [inline ips/src/lib.rs:39]: Same as above and also put a space between the `/` and the comment

File: ips/src/lib.rs [hunk 4]
```diff
@@ -23,96 +23,124 @@
...
-};
-use frame_system::ensure_signed;
-use sp_runtime::RuntimeDebug;
+// use frame_support::{
+// 	codec::{Decode, Encode},
+// 	decl_event, decl_module, decl_storage,
+// 	dispatch::DispatchResult,
+// };
+// use frame_system::ensure_signed;
+// use sp_runtime::RuntimeDebug;
```
Comment (arrudagates) [inline ips/src/lib.rs:52]: If this won't be used, remove the lines entirely

File: ips/src/lib.rs [hunk 5]
```diff
@@ -23,96 +23,124 @@
...
+
+        // #[pallet::weight(50_000 + T::DbWeight::get().writes(1))]
+        // pub fn send(origin: OriginFor<T>, ips: u32) -> DispatchResult {
+        // 	// The code goes here
+        // }
+
+        // #[pallet::weight(50_000 + T::DbWeight::get().writes(1))]
+        // pub fn destroy(origin: OriginFor<T>, ips: u32) -> DispatchResult {
+        // 	// The code goes here
+        // }
```
Comment (arrudagates) [inline ips/src/lib.rs:144]: Same as above

**Review by arrudagates — DISMISSED** [review]
Code looks good, just some stylistic details, also make sure to use `rustfmt` and `cargo clippy` on it

**Comment by SHA888** [comment]
@arrudagates I've made some changes accordingly

**Review by Invarchitect — APPROVED** [review]
Looks good, Kresna 👍

**Comment by SHA888** [comment]
I merge it for now. And then we could make any changes before or after the next meeting.

**Pull request merged**